src/components/Lightbox.tsx

```diff
@@ -52,6 +60,15 @@ const getLightboxFooterComponent =
         )}
         {content && <View style={styles.footerItem}>{content}</View>}
       </View>
+      {onPressDelete && (
```

Comment:
@SjaakSchilperoort we hadden het eerst over om de button(s) om de mogelijk aanwezige content te plaatsen, maar ik zag dat de buttons in het design een redelijk grote margin hebben heb ik dit maar onderaan gezet.
Een voordeel is wel dat er geen snapshot changes zijn voor componenten die de `onPressDelete` niet meegeven.